src/components/dashboard/SendTokens/BitcoinLikeBockchain/FormContainer.jsx

```diff
@@ -15,17 +14,16 @@ import {
   MODE_SIMPLE,
 } from 'components/constants'
 
-const mapStateToProps = (state, props) => ({
+export const mapStateToProps = (state, props) => ({
   token: state.get(DUCK_TOKENS).item(props.tokenSymbol)
 })
 
-function mapDispatchToProps (dispatch) {
+export function mapDispatchToProps (dispatch) {
   return {
     mainTransfer: (wallet, token, amount, recipient, feeMultiplier, advancedModeParams) => dispatch(mainTransfer(wallet, token, amount, recipient, feeMultiplier, advancedModeParams)),
   }
 }
 
-@connect(mapStateToProps, mapDispatchToProps)
```

Comment:
?!Why?

Reply:
I don’t understand the question. Please, can you explain what you asked about?